product_docs/docs/eprs/7/01_introduction/03_certified_supported_versions.mdx

```diff
@@ -16,7 +16,7 @@ You can use the following database product versions with Replication Server:
 
 All PostgreSQL and EDB Postgres Advanced Server versions available as EDB Postgres AI Cloud Service single-node and primary/standby high-availability cluster types are also supported for SMR configurations. See the Cloud Service [documentation](/edb-postgres-ai/cloud-service/) for more information about Cloud Service’s [supported cluster types](/edb-postgres-ai/cloud-service/references/supported_cluster_types/). See the [database version policy documentation](/edb-postgres-ai/cloud-service/references/supported_database_versions/) for the versions of PostgreSQL and EDB Postgres Advanced Server available in Cloud Service.
 
-EDB Postgres Distributed (PGD) v5.3.0 is explicitly certified as a Publishing database for trigger mode and as Subscription database for both trigger and wal modes.
+EDB Postgres Distributed (PGD) v5.6.0 is explicitly certified as a Publishing database for trigger mode and as Subscription database for both trigger and wal modes.
```

Comment:
@mlewandowski-edb @zahid75 
For PGD, we currently are at 5.6. 
Have we certified EPRS with 5.6 or any version later than 5.3? 

If not, I can roll back this change.